wrench/nodes/default.lua

```diff
@@ -0,0 +1,45 @@
+
+-- Register nodes from default / minetest_game
+
+wrench:register_node("default:chest", {
+	lists = {"main"},
+})
+
+wrench:register_node("default:chest_locked", {
+	lists = {"main"},
+	metas = {
+		owner = wrench.META_TYPE_STRING,
+		infotext = wrench.META_TYPE_STRING
+	},
+	owned = true,
+})
+
+wrench:register_node("default:furnace", {
+	lists = {"fuel", "src", "dst"},
+	metas = {
+		infotext = wrench.META_TYPE_STRING,
+		fuel_totaltime = wrench.META_TYPE_FLOAT,
+		fuel_time = wrench.META_TYPE_FLOAT,
+		src_totaltime = wrench.META_TYPE_FLOAT,
+		src_time = wrench.META_TYPE_FLOAT
+	},
+})
+
+wrench:register_node("default:furnace_active", {
+	lists = {"fuel", "src", "dst"},
+	metas = {
+		infotext = wrench.META_TYPE_STRING,
+		fuel_totaltime = wrench.META_TYPE_FLOAT,
+		fuel_time = wrench.META_TYPE_FLOAT,
+		src_totaltime = wrench.META_TYPE_FLOAT,
+		src_time = wrench.META_TYPE_FLOAT
+	},
+	store_meta_always = true,
+})
+
+wrench:register_node("default:sign_wall", {
+	metas = {
+		infotext = wrench.META_TYPE_STRING,
+		text = wrench.META_TYPE_STRING
+	},
+})
```

Comment:
I just noticed this node doesn't exist anymore, it's now `default:sign_wall_wood` and `default:sign_wall_steel`.

Reply:
Does this correctly give warning about it? https://github.com/mt-mods/technic/pull/242/files#diff-d43ef098b9f5fc8ee6223a94b85789cf9d7d19e3118d095217f1e55511edaa6cR33-R37

Reply:
Seems like it is not logging warnings about that, probably alias.
Maybe it could be changed anyway now that there's already other changes from #243